lms/djangoapps/courseware/user_state_client.py

```diff
@@ -250,6 +252,13 @@ def delete_many(self, username, block_keys, scope=Scope.user_state, fields=None)
         if scope != Scope.user_state:
             raise ValueError("Only Scope.user_state is supported")
 
+        if fields is None:
+            self._ddog_increment(evt_time, 'delete_many.empty_state')
```

Comment:
Need to define `evt_time`. I didn't do this in the helper method because all events from the same API call should have the same time.